config/webpack/webpack.config.js

```diff
@@ -131,23 +141,17 @@ const buildWebpackConfigs = builds.map(
 
     const internalJs = [paths.src, ...paths.compilePackages];
 
-    const entry = [paths.clientEntry];
-    const devServerEntries = [
-      `${require.resolve(
-        'webpack-dev-server/client'
-      )}?http://localhost:${port}/`
-    ];
-
-    if (args.script === 'start') {
-      entry.unshift(...devServerEntries);
-    }
-
     return [
       {
-        entry,
+        entry: {
+          client: paths.clientEntry,
```

Comment:
Interesting refactor here. Definitely easier to read, makes the difference between static and server-side rendering more pronounced unfortunately.

Reply:
I wouldn't be surprised if we should make the same changes to SSR, but as I haven't used sku for SSR so it's hard to test.